templates/donate.html

```diff
@@ -47,7 +47,10 @@ <h1>Donate Bitcoin</h>
             <form id="pay" action='/pay'>
                 <div style="display:block;text-align: center;">
                     <h2 style="margin:0;">Amount:
-                    <input id="amountenter" style="display:inline" size="4" type="float" name="amount" id="amount" placeholder="{{ params.currency }}" required>
+                    <input id="amountenter" style="display:inline" size="4" type="float" name="amount" id="amount" placeholder="{{ 0 }}" required>
+                    </h2>
+                    <h2 style="margin:0;">Currency:
+                    <select name="currency" id="currency" onclick="display_currencies();" required></select>
```

Comment:
This will also not work, because `static/satsale.js` is not included from `donate.html`. Also, JS is unnecessary here, everything can be done in a Python code before, look how `params["currency"]` was set in `satsale.py` before.

Reply:
Is this okay? I can't properly test I'm afraid, as previously discussed (apologies)